sn_cli/src/subcommands/wallet/helpers.rs

```diff
@@ -6,13 +6,48 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
+#[cfg(feature = "distribution")]
+use super::WalletApiHelper;
+#[cfg(feature = "distribution")]
+use base64::Engine;
 use color_eyre::Result;
 use sn_client::Client;
 use sn_transfers::{HotWallet, SpendAddress, Transfer};
 use std::{path::Path, str::FromStr};
 use url::Url;
 
-pub async fn get_faucet(root_dir: &Path, client: &Client, url: String) -> Result<()> {
+#[cfg(feature = "distribution")]
+pub async fn get_faucet(
+    root_dir: &Path,
+    client: &Client,
+    url: String,
+    address: Option<String>,
+    signature: Option<String>,
+) -> Result<()> {
+    if address.is_some() && signature.is_some() {
```

Comment:
We want to avoid `expect` in there, I know the outer check should make that fine, but if we were refactoring down the line this could open a can of `panic` worms instead of graceful error handling.

It's preferred to 
```rust
let addr = if let Some(addr) = address {
addr } 
else {
return Error(<SomeClearError> };

```
and handle that case explicitly with an error we can handle here/ elsewhere, rather than potentially panicking.

same for `sig`

Reply:
Thanks for the tip, always nice to get advice that removes potential panics. Fixed in https://github.com/maidsafe/safe_network/commit/34ea9e872e76ecce64ba7ba82492524a0a49a5ff